fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/cache/IdNameCache.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.cache;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import com.ibm.fhir.persistence.jdbc.dao.api.IIdNameCache;
+
+
+/**
+ * @param <T> the type of the key value held by the cache
+ */
+public class IdNameCache<T> implements IIdNameCache<T> {
+
+    // We use LinkedHashMap for the local map because we also need to maintain order
+    // of insertion to make sure we have correct LRU behavior when updating the shared cache
```

Comment:
I don't think this comment still applies to this implementation, which caches everything and doesn't need to be LRU

Reply:
comment removed